Sources/BaseDestination.swift

```diff
@@ -309,8 +321,11 @@ open class BaseDestination: Hashable, Equatable {
         case .error:
             color = levelColor.error
 
-        default:
-            color = levelColor.verbose
+        case .critical:
+            color = levelColor.critical
+            
+        case .fault:
```

Comment:
what means fault? is it the highest severity?

Reply:
Critical should be the highest level, fault is just below critical. The official documentation from Apple is here https://developer.apple.com/documentation/os/logger.